bot/cli/note_handlers.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module provides functions to manage notes using NoteBook, Note,
+Name, and Phone classes.
+
+Functions:
+- add_note(args: List[str], note_book: NoteBook) -> str:
+  Adds a new note with the given name and phone number to the Note book.
+
+Usage:
+This module can be imported and used in other Python scripts to manage a collection
+of notes. Each function handles specific operations related to adding, updating,
+and retrieving note information.
+"""
+
+from typing import List
+from bot.models import NoteBook, Note
+from bot.cli.input_error import input_error
+
+# @input_error
+def add_note(args: List[str], note_book: NoteBook) -> str:
```

Comment:
Implement a check to prevent saving an empty note.
If the note text is empty, display a message like "No note text provided."
Also, provide a usage example, such as "Usage: add-note <note text>" to guide the user.

Reply:
done